staging/src/k8s.io/apiserver/pkg/storage/etcd3/metrics/metrics.go

```diff
@@ -113,6 +113,14 @@ var (
 		},
 		[]string{"resource"},
 	)
+	decodeErrorCounts = compbasemetrics.NewCounterVec(
+		&compbasemetrics.CounterOpts{
+			Name:           "storage_decode_errors",
```

Comment:
The metric should be prefixed by `apiserver`

Reply:
Got it. Will rename to `apiserver_storage_decode_errors`